### ISSUE TITLE
Add cents under "area" in "units" popup 

### ISSUE THREAD

**Issue by kamathln** [j4velin/MapsMeasure#6]
Along with current hectares et al., also add "cents" (generally used in India)
(1 cent = 40.4685642 square meters)

**Comment by kamathln** [comment]
Preferebly allow to change units in the overlaid display.

**Comment by j4velin** [comment]
Never heard of that unit before... Is that used anywhere else except India?

**Comment by kamathln** [comment]
Not really. It is very specific to India. But was wondering if you could add a table with unit conversion data, and in settings, we could choose what gets shown. Units which are generally not used could be hidden.
